ui/src/hooks/useKubernetes.ts

```diff
@@ -769,7 +781,10 @@ export function createTerminal(el: HTMLElement, socket: WebSocket, lunaConfig: I
   initCustomWindowEvent(fitAddon);
   initMittBusEvents(searchAddon, socket);
 
-  return terminal;
+  return {
+    terminal,
+    searchAddon,
+  };
 }
 
 export function useKubernetes(t: any) {
```

Comment:
I'm sorry for the misunderstanding, but it seems like there is no code provided to analyze. Could you please share the code that needs to be checked?